tests/pybullet/test_diagnostics.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import os
+import pathlib
+import tempfile
+
+import gym
+import hydra
+import numpy as np
+import pybulletgym  # register PyBullet enviroments with open ai gym
+import torch
+import yaml
+from omegaconf import OmegaConf
+
+import mbrl.diagnostics as diagnostics
+import mbrl.planning as planning
+import mbrl.util.common
+
+_REPO_DIR = pathlib.Path(os.getcwd())
+_DIR = tempfile.TemporaryDirectory()
+_HYDRA_DIR = pathlib.Path(_DIR.name) / ".hydra"
+pathlib.Path.mkdir(_HYDRA_DIR)
+
+# Environment information
+_ENV_NAME = "HopperPyBulletEnv-v0"
+_ENV = gym.make(_ENV_NAME)
+_OBS_SHAPE = _ENV.observation_space.shape
+_ACT_SHAPE = _ENV.action_space.shape
+_CONF_DIR = pathlib.Path("mbrl") / "examples" / "conf"
+
+# Creating config files
+with open(
+    _REPO_DIR / _CONF_DIR / "dynamics_model" / "gaussian_mlp_ensemble.yaml",
+    "r",
+) as f:
+    _MODEL_CFG = yaml.safe_load(f)
+
+_CFG_DICT = {
+    "algorithm": {
+        "learned_rewards": True,
+        "target_is_delta": True,
+        "normalize": True,
+        "dataset_size": 128,
+    },
+    "dynamics_model": _MODEL_CFG,
+    "overrides": {
+        "env": f"gym___{_ENV_NAME}",
+        "term_fn": "no_termination",
+        "model_batch_size": 32,
+        "validation_ratio": 0.1,
+        "num_elites": 5,
+    },
+    "device": "cuda:0" if torch.cuda.is_available() else "cpu",
+}
+
+# Config file for loading a pytorch_sac agent
+with open(_REPO_DIR / _CONF_DIR / "algorithm" / "mbpo.yaml", "r") as f:
+    _MBPO__ALGO_CFG = yaml.safe_load(f)
+_MBPO_CFG_DICT = _CFG_DICT.copy()
+_MBPO_CFG_DICT["algorithm"] = _MBPO__ALGO_CFG
+_MBPO_CFG_DICT["overrides"].update(
+    {
+        "sac_alpha_lr": 3e-4,
+        "sac_actor_lr": 3e-4,
+        "sac_actor_update_frequency": 4,
+        "sac_critic_lr": 3.7e-5,
+        "sac_critic_target_update_frequency": 16,
+        "sac_target_entropy": -3,
+        "sac_hidden_depth": 2,
+        "num_steps": 2000,
+        "cem_elite_ratio": 0.1,
+        "cem_population_size": 500,
+        "cem_num_iters": 5,
+        "cem_alpha": 0.1,
+        "cem_clipped_normal": False,
+    }
+)
+
+# Extend default config file with information for a trajectory optimizer agent
+with open(_REPO_DIR / _CONF_DIR / "algorithm" / "pets.yaml", "r") as f:
+    _PETS_ALGO_CFG = yaml.safe_load(f)
+with open(_REPO_DIR / _CONF_DIR / "action_optimizer" / "cem.yaml", "r") as f:
+    _CEM_CFG = yaml.safe_load(f)
+_CFG_DICT["algorithm"].update(_PETS_ALGO_CFG)
+_CFG_DICT["algorithm"]["learned_rewards"] = True
+_CFG_DICT["algorithm"]["agent"]["verbose"] = False
+_CFG_DICT["action_optimizer"] = _CEM_CFG
+_CFG_DICT["seed"] = 0
```

Comment:
This is fine for a test, but I'm curious as to why are you getting this error, since the mujoco test doesn't need it, even when I download your branch. Which version of Hydra are you using?

Reply:
I am using hydra 1.0.3:
```
(mbrl-lib) dtch1997@desktop:~/code/mbrl-lib$ python
Python 3.8.11 (default, Aug  3 2021, 15:09:35)
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import hydra
>>> hydra.__version__
'1.0.3'
```